events/wc_plague_events.txt

```diff
@@ -89,6 +89,9 @@ narrative_event = {
 					}
 				}
 			}
+			c_46055 = {
+				narrative_event = { id = WCDRU.304 days = 365 }
```

Comment:
> 1 year

Dalaran can be already Scourged when this events fire

<details>

![image](https://user-images.githubusercontent.com/46576860/64396343-0162bb00-d066-11e9-98d9-9e4bc8aa32ea.png)

</details>